src/main/java/org/jenkinsci/plugins/fodupload/PayloadPackaging.java

```diff
@@ -34,7 +34,7 @@ public static PayloadPackaging getInstance(SastJobModel model, String technology
 
 final class PayloadPackagingImpl {
     static FilePath performPackaging(SastJobModel model, String technologyStack, Boolean openSourceAnalysis, String globalSCPath, FilePath workspace, PrintStream logger) throws IOException {
-        logger.println("Starting ScanCentral Packaging for source @ " + model.getSrcLocation());
+        logger.println("Starting Zip Packaging for source @ " + model.getSrcLocation());
```

Comment:
Move this to line 42, since that the if branch that does zip, or else if a ScanCentral package is running it'll log both messages